lib/wallet/src/Cardano/Wallet/Read/Primitive/Tx/Babbage.hs

```diff
@@ -83,6 +83,8 @@ import qualified Cardano.Wallet.Primitive.Types as W
 import qualified Cardano.Wallet.Primitive.Types.Coin as W
 import qualified Cardano.Wallet.Primitive.Types.Hash as W
 import qualified Cardano.Wallet.Primitive.Types.Tx as W
+import qualified Cardano.Wallet.Primitive.Types.Tx.TxOut as W
+    ( TxOut (TxOut) )
```

Comment:
is this avoidable ? the export list for a qualified import ?

Reply:
This allows people to write `W.TxOut` (which is a **_type_**), but prevents people from using the `W` qualifier for **_functions_** exported from the `TxOut` module.

For example, to use the `TxOut.mapAssetIds` function, we can write `W.TxOut.mapAssetIds`, but we're prevented from writing `W.mapAssetIds`. (The `Tx` module also has a `mapAssetIds` function, so it's desirable to discriminate between them.)